internal/tracing/manifests/values.go

```diff
@@ -0,0 +1,39 @@
+package manifests
+
+import (
+	"encoding/json"
+
+	"k8s.io/klog"
+)
+
+const (
+	subscriptionChannel = "stable"
+)
+
+type TracingValues struct {
+	Enabled                 bool   `json:"enabled"`
+	OTELSubscriptionChannel string `json:"otelSubscriptionChannel"`
+	OTELColSpec             string `json:"otelColSpec"`
+}
+
+func BuildValues(opts Options) (TracingValues, error) {
+	values := TracingValues{
+		Enabled:                 true,
+		OTELSubscriptionChannel: subscriptionChannel,
```

Comment:
Currently this is not configurable by the tracing helm chart, so you need to update the chart to support this configuration

Reply:
I don't want to configure it but keep the value in a variable to make it visible. We currently only support `stable`.